src/Symfony/Component/Validator/Mapping/ClassMetadata.php

```diff
@@ -217,7 +217,24 @@ public function getDefaultGroup()
     }
 
     /**
-     * {@inheritdoc}
+     * Adds a class constraint.
```

Comment:
Actually, the comment for ClassMetadata is not exactly the same as it for the GenericMetadata because for class meta data, its traversal strategy is affected by a `Traverse` constraint, while for generic meta data, `Valid` constraint should be used.